ext/winevt/winevt_channel.c

```diff
@@ -133,6 +133,7 @@ DWORD is_subscribable_channel_p(EVT_HANDLE hChannel, BOOL force_enumerate)
           status = GetLastError();
         } else {
           free(pProperty);
+          pProperty = NULL;
```

Comment:
```suggestion
```

`free(pProperty)` is done in `cleanup` label, so just remove `free()` here seems enough to fix.

Reply:
Looks for me that it is necessary when normal processing is done.

https://github.com/fluent-plugins-nursery/winevt_c/blob/dde5d46e3fc7439550deabae1208f321ae9065c4/ext/winevt/winevt_channel.c#L128-L133